perf/benchmark/README.md

```diff
@@ -71,18 +73,20 @@ The test has three sidecar modes:
 python runner/runner.py <conn> <qps> <duration> --OPTIONAL-FLAGS
 ```
 
-Where:
+Required fields:
 - `conn` = number of concurrent connections 
 - `qps` = queries per second for each connection 
 - `duration` = number of seconds to run each test for  (min: 92 seconds)
 
 Optional flags:
 
 ```
+  -h, --help          show this help message and exit
   --size SIZE         size of the payload
   --client CLIENT     where to run the test from
   --server SERVER     pod ip of the server
   --perf              also run perf and produce flamegraph
+  --linkerd LINKERD   linkerd mode
```

Comment:
Should it just be ` --linkerd    ...   linkerd mode` or does it take an argument?

Reply:
it doesn't take an argument, it's a bool. https://github.com/askmeegs/tools/blob/linkerd-install/perf/benchmark/runner/runner.py#L238 

should I change the format of the flag?